rtc-video-quality/generate_data.py

```diff
@@ -372,9 +372,9 @@ def decode_file(job, temp_dir, encoded_file):
   with open(os.devnull, 'w') as devnull:
     if job['codec'] in ['av1', 'vp8', 'vp9']:
       decoder = 'aom/aomdec' if job['codec'] == 'av1' else 'libvpx/vpxdec'
-      subprocess.check_call([decoder, '--i420', '--codec=%s' % job['codec'], '-o', decoded_file, encoded_file, '--framestats=%s' % framestats_file], stdout=devnull, stderr=devnull)
+      subprocess.check_call([decoder, '--i420', '--codec=%s' % job['codec'], '-o', decoded_file, encoded_file, '--framestats=%s' % framestats_file], stdout=devnull, stderr=devnull, encoding='utf-8')
```

Comment:
This line is too long. The Google Python style guide requires lines to be limited to 80 characters. We don't necessarily need to do that, but lines should be short enough that they don't get cut off when viewed in GitHub.
Please split it.

Reply:
Got it. There are a lot of lines that are too long from before that are too long. Should I make changes to those too ?